benchmark/bls.cpp

```diff
@@ -122,11 +148,11 @@ bls12_381::g2 random_g2()
 }
 
 // bls_g1_add benchmarking
-void benchmark_bls_g1_add() {
+void benchmark_bls_g1_add_impl(std::string test_name, bool mont) {
```

Comment:
In this and other `_impl` instances, change `test_name` to  `const &` as it is not moved in the function.